components/artist_page_cta/templates/register.jade

```diff
@@ -1,6 +1,6 @@
 div.artist-page-cta-overlay__register
   form.auth-form(
-    action=sd.AP.loginPagePath
+    action="#{sd.AP.loginPagePath}?modal_id=artist_page_cta"
```

Comment:
If you pass a query param of `modal_id`, [artsy-passport](https://github.com/artsy/artsy-passport/blob/c22154cf6f7dc422bd3bc80961f87b8c4e3abc9a/lib/app/analytics.coffee#L6) includes it in server-side tracking (what @anipetrov wants).